specification/azure-kusto/resource-manager/Microsoft.Kusto/stable/2019-01-21/examples/KustoClustersDelete.json

```diff
@@ -0,0 +1,13 @@
+﻿{
+  "parameters": {
+    "subscriptionId": "12345678-1234-1234-1234-123456789098",
+    "resourceGroupName": "kustorptest",
+    "clusterName": "KustoClusterRPTest4",
+    "api-version": "2019-01-21"
+  },
+  "responses": {
+    "200": {},
+    "202": {},

```

Comment:
update required.. for ex : for 202 location + retry-after headers

Reply:
I haven't seen other resource provider's examples with headers or anything in the result of delete operations. For instance EventGrid https://github.com/Azure/azure-rest-api-specs/blob/master/specification/eventgrid/resource-manager/Microsoft.EventGrid/stable/2019-01-01/examples/EventSubscriptions_DeleteForResource.json 

Do you have an example I can take a look at?